RevenueCat/Scripts/Utilities.cs

```diff
@@ -23,7 +23,7 @@ internal static DateTime FromUnixTimeInMilliseconds(long unixTimeInMilliseconds)
 
         internal static String DictToString<T>(Dictionary<string, T> dictionary)
         {
-            var items = dictionary.Select(kvp => string.Format($"{kvp.Key} : Id={kvp.Value.ToString()}"));
+            var items = dictionary.Select(kvp => $"{kvp.Key} : Id={kvp.Value.ToString()}");
```

Comment:
`string.Format` was causing an exception when printing an object with two nested dictionaries where the inner one was empty. Since we're not actually using the format functionality, instead we're interpolating the strings, we can safely remove it.

The code was effectively doing:

```
string.Format(string.Format("{{}}"));
```

Which throws:

```
FormatException: Input string was not in a correct format.
                                                                                                      at System.Text.StringBuilder.FormatError () [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at System.Text.StringBuilder.AppendFormatHelper (System.IFormatProvider provider, System.String format, System.ParamsArray args) [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at System.String.FormatHelper (System.IFormatProvider provider, System.String format, System.ParamsArray args) [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at System.String.Format (System.String format, System.Object[] args) [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at PurchasesListener.GetOfferings () [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at UnityEngine.Events.UnityEvent.Invoke () [0x00000] in <00000000000000000000000000000000>:0 
                                                                                                      at UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) [0x00000
```